include/swift/Basic/DemangleNodes.def

```diff
@@ -177,5 +177,7 @@ NODE(ThrowsAnnotation)
 NODE(EmptyList)
 NODE(FirstElementMarker)
 NODE(VariadicMarker)
+NODE(OutlinedEnumCopy)
+NODE(OutlinedEnumConsume)
```

Comment:
I would rename the nodes to OutlinedCopy/OutlinedConsume. The "enum" is already part of the type. We can then also reuse them for other types than enums.